### PR TITLE
Update Use Cases in the Developer Guide

### PR DESCRIPTION
**Pull request by Daphne789**
Add use cases for each FAPro feature: add, delete, find, edit, and help

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -287,32 +287,137 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
-      Use case resumes at step 2.
+        Use case resumes at step 2.
+
+**Use case: Find persons**
+
+**MSS**
+
+1.  Financial Advisor requests to list persons
+2.  FAPro shows a list of persons
+3.  Financial Advisor requests to find a specific persons in the list by name
```
Comment (limzhenwy) [inline docs/DeveloperGuide.md]: Possible typo? "persons" -> "person"

**Review by limzhenwy — COMMENTED** [review]
LGTM. Few nits to fix.

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (Daphne789) [reply docs/DeveloperGuide.md]: Screenshot 2023-10-03 130920

I follow the format of the current Developer Guide. So, should I change all persons -> person or just change the number 3?

Comment (bhnuka) [reply docs/DeveloperGuide.md]: Just number 3. The other uses of persons are fine.

Comment (Daphne789) [reply docs/DeveloperGuide.md]: Noted. Thank you @bhnuka @limzhenwy

**Comment by bhnuka** [comment]
LGTM. Thanks for the quick corrections.

**Pull request merged**